sdk/php/runtime/go.mod

```diff
@@ -38,3 +38,11 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240515191416-fc5f0ca64291 // indirect
 	google.golang.org/protobuf v1.34.1 // indirect
 )
+
+replace go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc => go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploggrpc v0.0.0-20240518090000-14441aefdf88
+
+replace go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploghttp => go.opentelemetry.io/otel/exporters/otlp/otlplog/otlploghttp v0.3.0
+
+replace go.opentelemetry.io/otel/log => go.opentelemetry.io/otel/log v0.3.0
+
+replace go.opentelemetry.io/otel/sdk/log => go.opentelemetry.io/otel/sdk/log v0.3.0
```

Comment:
What does this do? As I noticed they are the exact versions already specified above:

https://github.com/dagger/dagger/blob/8dabfd657d299904fab8be718e6b7d06409f31a6/sdk/php/runtime/go.mod#L10-L16

Reply:
Not sure about this change. WDYT @vito?

Reply:
This is from https://github.com/dagger/dagger/pull/8234

tl;dr: We still depend on some pre-1.0 versions of OTel SDK packages, and it has happened before that even though our `go.mod` initially pins them appropriately, a user running `go get -u` or similar will have them bump ahead to a version that breaks everything. So we explicitly add `replace` rules for them now.